pkgs/applications/networking/p2p/freenet/default.nix

```diff
@@ -38,8 +30,8 @@ let
       mkdir -p $out/share/freenet
       ln -s ${bcprov} $out/share/freenet/bcprov.jar
       ln -s ${freenet_ext} $out/share/freenet/freenet-ext.jar
-      ln -s ${jna_platform} $out/share/freenet/jna_platform.jar
-      ln -s ${jna} $out/share/freenet/jna.jar
+      ln -s ${jna}/share/java/jna-platform.jar $out/share/freenet/jna_platform.jar
```

Comment:
The same JNA works in both places? It looks unusual.

Reply:
This is linking in the `jna` so that the wrapper script down below can access it. There it will construct the java classpath out of this. I plan on removing this wrapper script eventually, but with this PR I am just swapping out where the `jna.jar` comes from.